src/index.html

```diff
@@ -6,9 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="./images/logo.png" 
+          class="logo" 
+          alt="Logo MOYO"
+        >
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li>
+            <a href="#apple" class="nav_link is-active ">
+              Apple
+            </a>
+          </li>
+          <li><a href="#samsung" class="nav_link">Samsung</a></li>
```

Comment:
I think this syntax will improve the readability of the code
```suggestion
          <li>
            <a href="#samsung" class="nav_link">Samsung</a>
          </li>
```